### PR TITLE
[4.3][Guided Tours] Fixed missing field in Banners Tour

### PR DESCRIPTION
**Pull request by khu5h1**
Pull Request for Issue #40210 .

### Summary of Changes
Fixed missing field in Banners Tour.

### Actual result BEFORE applying this Pull Request
When we go through the tutorial on how to create a banner, there are skipping fields, some of which are required.

### Expected result AFTER applying this Pull Request
Now, it does not skip the fields and targets the entire details.

image

### CODE REVIEW

File: administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+UPDATE `#__guidedtour_steps`
+SET `target` = '.col-lg-9'
+WHERE `target` = '#jform_description,#jform_description_ifr' AND `id` = 63;
```
Comment (richard67) [inline administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql]: Shouldn't also the title and the description be changed here and not only the target? In the other SQL scripts you change the language strings used for the target and the description, but here you don't to it when updating a 4.3.0-beta4 or 4.30-rc1.

File: administrator/components/com_admin/sql/updates/postgresql/4.3.0-2023-03-31.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+UPDATE "#__guidedtour_steps"
+SET "target" = '.col-lg-9'
+WHERE "target" = '#jform_description,#jform_description_ifr' AND "id" = 63;
```
Comment (richard67) [inline administrator/components/com_admin/sql/updates/postgresql/4.3.0-2023-03-31.sql]: Shouldn't also the title and the description be changed here and not only the target? In the other SQL scripts you change the language strings used for the target and the description, but here you don't to it when updating a 4.3.0-beta4 or 4.30-rc1.

In addition, please update a line feed to the end of the file.

File: administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql (hunk 1 above)
Comment (richard67) [reply administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql]: In addition, please update a line feed to the end of the file.

**Review by richard67 — CHANGES_REQUESTED** [review]

File: administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql (hunk 1 above)
Comment (khu5h1) [reply administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql]: @richard67 Yes, the title and description should also be changed. Will do the changes. Thanks!

Comment (richard67) [inline administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql]:
```suggestion
UPDATE `#__guidedtour_steps`
SET `title` = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_TITLE', `description` = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_DESCRIPTION', `target` = '.col-lg-9'
WHERE `id` = 63;

```

File: administrator/components/com_admin/sql/updates/postgresql/4.3.0-2023-03-31.sql (hunk 2 above)
Comment (richard67) [inline administrator/components/com_admin/sql/updates/postgresql/4.3.0-2023-03-31.sql]:
```suggestion
UPDATE "#__guidedtour_steps"
SET "title" = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_TITLE', "description" = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_DESCRIPTION', "target" = '.col-lg-9'
WHERE "id" = 63;

```

**Comment by richard67** [comment]
@khu5h1 Please do it like I suggest, update the values with one common update statement and use only the ID in the WHERE clause:

```
UPDATE `#__guidedtour_steps`
SET `title` = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_TITLE', `description` = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_DESCRIPTION', `target` = '.col-lg-9'
WHERE `id` = 63;

```

File: administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql (hunk 1 above)
Comment (richard67) [reply administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql]: Please do it in the way I just suggested.

**Comment by khu5h1** [comment]
@richard67 sure, updating the changes.

File: administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+UPDATE `#__guidedtour_steps`
+SET `title` = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_TITLE', `description` = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_DESCRIPTION', `target` = '.col-lg-9'
+WHERE `id` = 63;
```
Comment (richard67) [inline administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql]:
```suggestion
UPDATE `#__guidedtour_steps`
SET `title` = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_TITLE', `description` = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_DESCRIPTION', `target` = '.col-lg-9'
WHERE `id` = 63;

```

File: administrator/components/com_admin/sql/updates/postgresql/4.3.0-2023-03-31.sql [hunk 4]
```diff
@@ -0,0 +1,3 @@
+UPDATE "#__guidedtour_steps"
+SET "title" = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_TITLE', "description" = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_DESCRIPTION', "target" = '.col-lg-9'
+WHERE "id" = 63;
```
Comment (richard67) [inline administrator/components/com_admin/sql/updates/postgresql/4.3.0-2023-03-31.sql]:
```suggestion
UPDATE "#__guidedtour_steps"
SET "title" = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_TITLE', "description" = 'COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DETAILS_DESCRIPTION', "target" = '.col-lg-9'
WHERE "id" = 63;

```

File: administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql (hunk 3 above)
Comment (richard67) [reply administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-31.sql]: Add empty line to the end.

File: administrator/components/com_admin/sql/updates/postgresql/4.3.0-2023-03-31.sql (hunk 4 above)
Comment (richard67) [reply administrator/components/com_admin/sql/updates/postgresql/4.3.0-2023-03-31.sql]: Add empty line to the end.

**Review by richard67 — APPROVED** [review]

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on 1e366a789c07c12e8f903a9d82d62659a9ce5765<br /><br />It's all right now, thanks. A little generalized (large) field with details, but it's better than it was ;).<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40258">issues.joomla.org/tracker/joomla-cms/40258</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1e366a789c07c12e8f903a9d82d62659a9ce5765<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40258">issues.joomla.org/tracker/joomla-cms/40258</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40258">issues.joomla.org/tracker/joomla-cms/40258</a>.</sub>

**Comment by sdwjoomla** [comment]
Thanks khu5h1

**Pull request merged**